src/main/native/include/sysid/analysis/FilteringUtils.h

```diff
@@ -98,7 +74,13 @@ units::second_t TrimStepVoltageData(std::vector<PreparedData>* data,
 
 /**
  * Compute the mean time delta of the given data.
+ *
+ * @param data A reference to all of the collected PreparedData
+ *
+ * @return The mean time delta for all the data points
  */
 units::second_t GetMeanTimeDelta(const Storage& data);
 
+void FilterAccelData(std::vector<PreparedData>* data);
```

Comment:
This needs a Doxygen comment explaining what "filter accel" means in this context.